tests/lib.rs

```diff
@@ -10,6 +10,8 @@ use predicates::boolean::PredicateBooleanExt;
 use predicates::prelude::predicate::str::{contains, is_empty, similar};
 use tempfile::{Builder, TempDir};
 
+static CACHE_DIR: &str = "TEALDEER_CACHE_DIR";
```

Comment:
Same here for the name

Reply:
Renamed. Also added a TODO that this can be taken from `cache::CACHE_DIR_ENV_VAR` some day.

Reply:
Good point 👍